package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@la-ots/pelican",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
This is technically only changing documentation, so it shouldn't bump the version number.

Reply:
will do @james-alt